event/src/app/login/page.tsx

```diff
@@ -52,12 +52,12 @@ export default function LoginPage() {
           <Input type="password" name="password" id="password" />
           <button
             type="submit"
-            className="w-full rounded-md bg-blue-500 px-4 py-2 text-white hover:bg-blue-600 focus:bg-blue-600 focus:outline-none"
+            className="w-full rounded-md bg-gradient-to-r from-red-500 to-blue-500 px-4 py-2 text-white hover:bg-blue-600 focus:bg-blue-600 focus:outline-none"
           >
             Login
           </button>
         </form>
-        <Link href="/register" className="mt-4 block text-blue-600 underline">
+        <Link href="/register" className="mt-4 block text-blue-600 ">
```

Comment:
Suggest to add `underline` to indicate it is a link and use `w-fit` so the link doesn't span the entire block. To also do the same on the `/register` page (add `w-fit`).
```suggestion
        <Link href="/register" className="mt-4 block w-fit text-blue-600 underline">
```